service/src/main/java/com/softserve/osbb/service/RegistrationService.java

```diff
@@ -9,7 +9,10 @@
  */
 @Service
 public interface RegistrationService {
+	
+	StringBuilder getPassword();
```

Comment:
Not good.

Password should be of type String. If you are constructing password with StringBuilder - just call toString at the end.